docs/source/quickstart/index.rst

```diff
@@ -1,6 +1,30 @@
 Quickstart
 ==========
 
-(In the future, this section will describe how to use the `OpenDP Library`_.)
+.. contents:: |toctitle|
+    :local:
 
-.. _OpenDP Library: https://github.com/opendp/opendp
+Installation
+------------
+
+The easiest way to get started with OpenDP is from Python. Use ``pip`` to install the `opendp <https://pypi.org/project/opendp/>`_ package from PyPI.
+
+.. code-block:: bash
+
+    % pip install opendp
+
+This will make the OpenDP modules available to your local environment.
+
+Hello, OpenDP!
+--------------
+
+Once you've installed OpenDP, you can write your first program. In the example below, we'll construct an identity ``Transformation``, then invoke it on a string.
+
+.. doctest::
+
+    >>> from opendp.trans import make_identity
+    >>> from opendp.typing import SubstituteDistance
+
+    >>> identity = make_identity(M=SubstituteDistance, TA=str)
```

Comment:
The identity transformation seems like a simple transform, but the nature of the function is so unconstrained that it has many free variables. I'd also say we generally work with SymmetricDistance, so using it in the quickstart may be confusing. The identity function also doesn't have much of a use. Perhaps we could switch to clamp?